.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: 'type: bug'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## First of all
+        1. Please search for [open issues](https://www.github.com/tauri-apps/tauri/issues) about this problem first.
+        2. Make sure `rustc` and all relevant packages are up to date.
+        3. Make sure it's not an issue with something else you are using.
```

Comment:
```suggestion
        3. Make sure it's an issue with Tauri and not something else you are using.
```

Reply:
Personally I think it's more readable as it is (and it's ofc shorter)

Reply:
Shorter doesn't necessary mean better, and I think we need to be explicit about this.